bagitobjecttransfer/recordtransfer/templates/includes/messages.html

```diff
@@ -0,0 +1,9 @@
+{% if messages %}
+    <div class="messages">
+        {% for message in messages %}
+            <div class="message {{ message.tags }}">
+                {{ message }}
+            </div>
+        {% endfor %}
+    </div>
+{% endif %}
```

Comment:
Is this file being used? If not, it can be deleted.

Reply:
Yes, this file is being used to display the messages